web-app/js/portal/prototypes/OpenLayers.js

```diff
@@ -250,6 +250,103 @@ OpenLayers.Geometry.prototype.isBox = function() {
 };
 
 OpenLayers.Geometry.prototype.toWkt = function() {
-    var wktFormatter = new OpenLayers.Format.WKT();
+    var wktFormatter = new OpenLayers.Format.WKTemii();
     return wktFormatter.write({ geometry: this });
 };
+
+OpenLayers.Geometry.prototype.crossesDateLine = function() {
+    var bounds = this.getBounds();
+    return (normaliseLongitude(bounds.left) > normaliseLongitude(bounds.right));
+};
+
+/**
+ * Class: OpenLayers.Format.WKTemii
+ * Openlayers 2.13.1 is creating geometries (for EPSG:4326) with longitude sometimes represented > 180
+ * WKTemii writes points with longitude normalized to between -180 to 180 */
+
+OpenLayers.Format.WKTemii = OpenLayers.Class(OpenLayers.Format.WKT, {
```

Comment:
This class can have a better name. It's not about emii, it's about having a projection of -180 to 180. Correct me if I'm wrong.